src/Akka.Streams.Kafka/Messages/Committable.cs

```diff
@@ -44,7 +45,7 @@ public interface ICommittableOffsetBatch : ICommittable
     /// <summary>
     /// Get current offset positions
     /// </summary>
-    IImmutableSet<GroupTopicPartitionOffset> Offsets { get; }
+    IImmutableDictionary<GroupTopicPartition, Offset> Offsets { get; }
```

Comment:
BREAKING API CHANGE

an `ImmutableHashSet` of `GroupTopicPartitionOffset` does not work here because we might have multiple competing offsets for a single partition depending on how the commit batching goes. We want a single dictionary per `GroupTopicPartition` instead.

I hate to make a breaking change but this one was bad.